tasks/tasks.py

```diff
@@ -15,10 +15,7 @@
 def with_vagrant(func):
```

Comment:
Since this function's scope is starting to grow, I'd prefer to move the whole `with_vagrant` decorator to `tasks/vagrant.py`